src/main/webapp/app/upload/upload.component.html

```diff
@@ -26,9 +26,95 @@ <h1 class="col-md-12" jhiTranslate="upload.title">Upload file with pools</h1>
         </div>
+
+        <button class="btn btn-primary" (click)="onSubmit()"
+                type="submit" jhiTranslate="upload.upload">
+            Upload
+        </button>
+        <button (click)="onForceSave()" class="btn btn-primary"
+                jhiTranslate="upload.forceSave" type="submit">
+            Force Save
+        </button>
+        <h1 class="col-md-12" jhiTranslate="upload.conflict">Conflicts</h1>
+        <div class="table_list">
+
+            <table class="table_side_by_side">
+                <thead>
+                <tr class="row">
+                    <th class="col-md-3"><p jhiTranslate="pools.list.headers.name">Name</p></th>
+                    <th class="col-md-2"><p jhiTranslate="pools.list.headers.size">Size</p></th>
+                    <th class="col-md-1"><p jhiTranslate="pools.list.headers.enabled">On</p></th>
+                    <th class="col-md-5"><p jhiTranslate="pools.list.headers.tools">Tools</p></th>
+                    <th class="col-md-2"></th>
+                </tr>
+                </thead>
+
+                <tbody>
+                <tr class="row" *ngFor="let pool of currentConflictPool">
+                    <td class="col-md-3">
+                        {{pool.displayName}}
+                    </td>
+                    <td class="col-md-2">{{pool.maximumCount}}</td>
+                    <td class="col-md-1">
+                        <fa-icon *ngIf="pool.enabled" icon="check"></fa-icon>
+                    </td>
+                    <td class="col-md-5">
+                        <div *ngFor="let tool of pool.tools" class="d-inline">
+                            <span>{{tool.name}} ({{tool.version}})</span>
+                        </div>
+                    </td>
+                    <td class="col-md-2">
+                        <label>
+                            <input type="radio" [name]="pool.displayName" (click)="choosePattern.set(pool.poolId,poolType.Old)">
+                        </label>
+                    </td>
+                </tr>
+                </tbody>
+            </table>
+
+            <table class="table_side_by_side">
+                <thead>
+                <tr class="row">
+                    <th class="col-md-2"></th>
+                    <th class="col-md-3"><p jhiTranslate="pools.list.headers.name">Name</p></th>
+                    <th class="col-md-2"><p jhiTranslate="pools.list.headers.size">Size</p></th>
+                    <th class="col-md-1"><p jhiTranslate="pools.list.headers.enabled">On</p></th>
+                    <th class="col-md-5"><p jhiTranslate="pools.list.headers.tools">Tools</p></th>
+                </tr>
+                </thead>
+
+                <tbody>
+                <tr class="row" *ngFor="let pool of newConflictPool">
+                    <td class="col-md-2">
+                        <label>
+                            <input type="radio" [name]="pool.displayName" (click)="choosePattern.set(pool.poolId,poolType.New)">
```

Comment:
Can we force one of the radio to be always checked ? Old one or new one doesn't matter, because when you try to save when some dots aren't choosen there is a `org.springframework.transaction.TransactionSystemException: Could not commit JPA transaction; nested exception is javax.persistence.RollbackException: Error while committing the transaction`

Reply:
Of course any other solution to this problem is good as well

Reply:
fixed, new things are now considered as default